src/flowr.ts

```diff
@@ -81,6 +84,8 @@ if(options['no-ansi']) {
 	setFormatter(voidFormatter)
 }
 
+if(options['config-file'])
```

Comment:
braces :sob: (:soap:)

but just as a quick check: does it works as well when someone gives relative and absolute paths (`../../.dotfiles/flowr.mega.config.json`, `/etc/trash/oldstuff/config`)?